src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,23 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.ActorDao;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(ActorDao actorDao) {
+        this.actorDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        return actorDao.get(id).orElse(new Actor());
```

Comment:
Why do you return an empty actor, if you couldn't find one by ID? You should throw an exception in that case. Same in other services' implementations.

Reply:
between returning an empty object or throwing an exception I choose here first because I don`t have any requirements about this in README or Common mistake, it is almost the end of the program, and if it will return an empty object I just will see this in main. And tests go normally so I don`t think that it matters.

Reply:
But you should throw an exception. How can users understand that they don't have this row in the database?
fix it in all places